lib/util.js

```diff
@@ -1057,3 +1057,55 @@ exports._exceptionWithHostPort = function(err,
 // process.versions needs a custom function as some values are lazy-evaluated.
 process.versions[exports.inspect.custom] =
   (depth) => exports.format(JSON.parse(JSON.stringify(process.versions)));
+
+
+// Callbackify section
+const CBFY_ERR = 'The last argument to the callbackified function must be a ' +
+                 'callback style function';
+
+
+function _cbfy_onRejected(rej, cb) {
```

Comment:
Can we rename this function to something like `function callbackifyOnRejected(reason, cb)`? The current function signature is sort of hard to read.